torchtune/datasets/_grammar.py

```diff
@@ -74,7 +76,13 @@ def grammar_dataset(
     ds = SFTDataset(
         source=source,
         message_transform=message_transform,
-        model_transform=model_transform,
+        model_transform=tokenizer,
```

Comment:
So we use model_transform in some places and tokenizer in others?

Reply:
See other comment